### PR TITLE
Add Vocabulary Input implementation instead of buefy input

### LINKED ISSUE #224: Use Vocabulary VInput component
## Description

This issue is part of the larger project of replacing third-party components with CC Vocabulary components. Buefy inputs (text inputs and textareas) need to be replaced with VInput from Vocabulary.

## Implementation

- [x] I would be interested in implementing this feature.

### LINKED ISSUE #259: Text style in the name of input component
## Description
The text style of the input name should have a different font-weight. This is a component change.

## Screenshots
Screen Shot 2020-12-16 at 11 36 01

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #224 by @obulat
Fixes #259 by @panchovm

## Description

This PR replaces a buefy input component with VInput component in the Attribution Details step. When this component is reviewed and merged into Vocabulary, Chooser will convert to that, so this is a temporary solution.

## Screenshots

Mobile: 
image

Tablet and up:
image

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/Vocabulary/VInput.vue [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+            default: ''
+        },
+        description: {
+            type: String,
+            default: ''
+        },
+        placeholder: {
+            type: String,
+            default: ''
+        },
```
Comment (dhruvkb) [inline src/Vocabulary/VInput.vue:71]: You can use `inheritAttrs: false` and then place the `$attrs` on the input. See [docs](https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance).

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: src/Vocabulary/VInput.vue (hunk 1 above)
Comment (obulat) [reply src/Vocabulary/VInput.vue:71]: Sorry for a late reply, @dhruvkb ! I was writing this component as a more general Input component for Vocabulary that can be a text input or a text area.  The solution you propose is really clean, but for a simpler component.

Some of these props, such as `description`, `label`, `value`  are necessary for the component itself, and some could be bound to the `input` or the `textarea`, so I thought leaving them as props would be better.

**Review by zackkrida — APPROVED** [review]

**Comment by zackkrida** [comment]
@obulat Some small merge conflicts; but otherwise this looks good!

**Comment by obulat** [comment]
With this PR merged, the size of `chunk-vendors.js` decreased from 649.41 KiB to 388.52 KiB (or from 187.05 KiB to 126.59 KiB gzipped), the sent file is now more than 30% smaller!

**Pull request merged**